terraform/aws/modules/customer-web-server/web-server.tf

```diff
@@ -0,0 +1,386 @@
+provider "aws" {
+  region = var.region
+}
+
+data "terraform_remote_state" "cluster" {
+  backend = "s3"
+  config = {
+    bucket = "terraform-cloud-monitoring-state-bucket-${var.environment}"
+    key    = "mattermost-central-command-control"
+    region = "us-east-1"
+  }
+}
+
+resource "aws_security_group" "cws_postgres_sg" {
+  name                   = "cws_postgres_sg"
+  description            = "postgres SG for customer web server"
+  vpc_id                 = var.vpc_id
+  revoke_rules_on_delete = true
+
+  ingress {
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    security_groups = [data.terraform_remote_state.cluster.outputs.workers_security_group]
+  }
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["10.247.4.47/32"]
```

Comment:
@angeloskyratzakos add a variable for this to avoid having a hardcoded VPN cidr.

Reply:
@stylianosrigas will update also that one: https://github.com/mattermost/mattermost-cloud-monitoring/blob/master/terraform/aws/modules/cluster-post-installation/provisioner-db.tf#L31

Reply:
Nice thanks